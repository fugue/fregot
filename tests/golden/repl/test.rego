package fregot.tests.repl.test

test_foo {
  1 + 1 > 1
}
