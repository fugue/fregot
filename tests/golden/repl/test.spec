{
    "command": "fregot",
    "arguments": ["repl", "--no-history-file"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${SPEC_NAME}.stderr"},
        {"stdout": "${SPEC_NAME}.stdout"}
    ],
    "stdin": [
        ":load test.rego",
        ":test"
    ]
}
