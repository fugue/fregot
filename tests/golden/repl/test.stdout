repl% repl% fregot.tests.repl.test% passed: 1, failed: 0, errored: 0
fregot.tests.repl.test% 